main.go

```diff
@@ -62,8 +65,15 @@ func main() {
 	log.Debugf("linode-region: %s", *linodeRegionParamPtr)
 	log.Debugf("linode-label: %s", *linodeLabelParamPtr)
 
+	//
+	docker, err := client.NewClient(fmt.Sprintf("unix://%s", *socketAddressParamPtr), "v1.18", nil, map[string]string{"User-Agent": "linode-volume-driver"})
```

Comment:
What is this bit about - and should this look more like what was done in #21?

Reply:
...as it compares to https://github.com/linode/docker-volume-linode/pull/22/files#diff-749da71b40f8ff06fc9e78ce917b0cceL233

Reply:
- That line is the docker client library.
- Are you talking about the User-Agent? 
- `unix://` what the driver uses to communicate with the docker daemon.
- you are right bout the version.... I'll look more into it.  we probably need to go as low as possible on the version number.

Reply:
- That line is the docker client library.
- Are you talking about the User-Agent? 
- `unix://` what the driver uses to communicate with the docker daemon.
- you are right bout the version.... I'll look more into it.  we probably need to go as low as possible on the version number.